lib/mes-adresse-api.js

```diff
@@ -1,7 +1,7 @@
 import HttpError from './http-error'
 import getConfig from 'next/config'
 
-const {NEXT_PUBLIC_BAL_API_URL: API_MES_ADRESSES_URL} = getConfig().publicRuntimeConfig
+const {NEXT_PUBLIC_BAL_API_URL: API_MES_ADRESSES_URL} = getConfig().publicRuntimeConfig || 'https://api-bal.adresse.data.gouv.fr/v2'
```

Comment:
On ne souhaite plus avoir de valeur par default dans le code.

```suggestion
const {NEXT_PUBLIC_BAL_API_URL: API_MES_ADRESSES_URL} = getConfig().publicRuntimeConfig
```

On a volontairement supprimer les valeur par défaut car on souhaite que l'app plante si la valeur n'est pas spécifier au sein des variables d'env. 

Ces variables sont inclues dans la config au démarrage de l'app (Et non plus seulement au build). Enfin, les valeurs présente dans le fichier `.env.default` sont chargées par défaut avant d'être surchargées par celle présente dans le fichier `.env`.